pkg/view/component/button.go

```diff
@@ -88,6 +161,20 @@ func NewButton(name string, payload action.Payload, options ...ButtonOption) But
 	return button
 }
 
+// NewButtonLink creates a button component that navigates to a link
+func NewButtonLink(name, ref string, options ...ButtonOption) Button {
```

Comment:
Not sure if I like the `NewButtonLink` .. it kind of confuses the issue of "this is a button with an action" and "this is a link that is a button". I'm willing to have my mind changed though ...